docs/src/main/asciidoc/sqs.adoc

```diff
@@ -268,3 +268,38 @@ public void receive(S3EventNotification s3EventNotificationRecord) {
 	S3EventNotification.S3Entity s3Entity = s3EventNotificationRecord.getRecords().get(0).getS3();
 }
 ----
+
+=== IAM Permissions
+Following IAM permissions are required by Spring Cloud AWS:
+
+[cols="2"]
+|===
+| Send message to Queue
+| `sqs:SendMessage`
+
+| Receive message from queue
+| `sqs:ReceiveMessage` and `sqs:DeleteMessage`
```

Comment:
wonder if we should add a new row for `sqs:DeleteMessage`

Reply:
True. It is better to go in another section.